### DISCUSSION TITLE
How to change font size in aterm?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#3524] [Q&A]
The font size in aterm is very small and I want to change it to the bigger size. How do I do it?
![Screenshot_20230725-174749~2](https://github.com/termux/termux-app/assets/129522477/efc8980d-46c4-4e69-a955-192e2f669f66)

**Comment by agnostic-apollo** [comment]
Pinch and zoom with two fingers.

**Reply by twaik** [reply]
Aterm is not termux-app, discussion should be moved to `termux-packages`.

**Reply by agnostic-apollo** [reply]
Sorry. But this is a question for aterm, not termux.

**Reply by ghost** [reply]
That's for the vnc viewer app, not aterm.
